homeassistant/components/insteon/config_flow.py

```diff
@@ -274,6 +279,8 @@ async def async_step_init(self, user_input=None):
         }
         if self.config_entry.data.get(CONF_HOST):
             data_schema[vol.Optional(STEP_CHANGE_HUB_CONFIG)] = bool
+        else:
+            data_schema[vol.Optional(STEP_CHANGE_PLM_CONFIG)] = bool
```

Comment:
This form would benefit from using the `show_menu` helper.

https://github.com/home-assistant/core/blob/5ff0479c16c121af2c79f28e4243c13daec7cd24/homeassistant/data_entry_flow.py#L555-L566